bedrock/firefox/templates/firefox/partners/index.html

```diff
@@ -459,7 +459,12 @@ <h2 class="tween">
         <div class="intro tween">
           {% if l10n_has_tag('mwc_2015') %}
             <h4>{{ _('A superior mobile experience') }}</h4>
-            <a href="{{ settings.GOOGLE_PLAY_FIREFOX_LINK }}"><img src="{{ static('img/firefox/partners/google-play.png') }}" alt="{{ _('Get it on Google Play') }}" class="play" /></a>
+            <div class="play-button"> 
+              <a href="{{ settings.GOOGLE_PLAY_FIREFOX_LINK }}"><img src="{{ static('img/firefox/partners/google-play.png') }}" alt="{{ _('Get it on Google Play') }}" class="play" /></a>
+              <small class="download-other">
+                <a href="{{ url('firefox.android.all') }}">{{_('Systems &amp; Languages')}}</a>
+              </small>
```

Comment:
The color contrast here is not good and is generally hard to read. 

![screen shot 2015-09-14 at 12 03 22](https://cloud.githubusercontent.com/assets/10350960/9847406/efc355aa-5ad8-11e5-9737-35b0fe09ea31.png)

I would suggest we make it:

```
.partner-article.dark a {
    color: white;
    font-size: .75rem;
}
```
![screen shot 2015-09-14 at 12 05 57](https://cloud.githubusercontent.com/assets/10350960/9847419/ff80c8f6-5ad8-11e5-83d6-c11796f322bd.png)